package-lock.json

```diff
@@ -3499,7 +3600,7 @@
       "dependencies": {
         "doctrine": {
           "version": "1.5.0",
-          "resolved": "https://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
+          "resolved": "http://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
```

Comment:
This leads me to suspect you generated this file using wrong version of Node. Did you install your dependencies using Node v12.7.0? (As described in `.nvmrc` file.)